taskfiles/rp.yml

```diff
@@ -31,6 +31,8 @@ tasks:
       CC: '{{.COMPILER}}'
       CXX: '{{if eq .COMPILER "gcc"}}g++{{else}}clang++{{end}}'
       ENABLE_GIT_VERSION: '{{default "ON" .ENABLE_GIT_VERSION}}'
+    deps:
+      - task: :dev:install-v8-build-tools
```

Comment:
can you just `apt-get install v8-libs-dev` or smth like that? why build it ourselves if general purpose OS will have that, no?

Reply:
https://github.com/vectorizedio/vtools/discussions/186

Not, packages for Fedora contains too old api

Reply:
i see.

Reply:
the OSS workflows only use ubuntu, so it does not need to work with fedora packages, no?

Reply:
ah even better.

Reply:
@dotnwat Can we support only ubuntu?

Reply:
Oh, it is my mistake
https://launchpad.net/ubuntu/+source/nodejs contains old api too. We have to add v8 to oss.cmake